### PR TITLE
Visitor can view unreviewed articles

### PR DESCRIPTION
**Pull request by stefankarlberg**
# [PT STORY](https://www.pivotaltracker.com/story/show/166284542)
## Changes we propose in this PR
* Tests with fixtures for new functionality  - listing of unreviewed files.
* When clicking on "Review articles" on landing page, one should be redirected to the Reviews page
* The visitor should be able to click on one article and see full article
* Refactoring and adjusting test files.

## What we have learned
* Debugging and more understanding of fixture responses.
* Cypress functionality

Screenshot 2019-05-29 at 17 48 05

Screenshot 2019-05-29 at 17 48 26

### CODE REVIEW

File: src/Components/ListOfArticles.jsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ class ListOfArticles extends Component {
     });
   }
 
+  
+
```
Comment (oliverochman) [inline src/Components/ListOfArticles.jsx]: Remove empty lines

File: src/index.js [hunk 2]
```diff
@@ -6,11 +6,11 @@ import * as serviceWorker from './serviceWorker';
 import { BrowserRouter } from 'react-router-dom'
 import axios from 'axios'
 
-axios.defaults.baseURL = 'https://glocal-news.herokuapp.com'
+axios.defaults.baseURL = 'http://localhost:3000'
```
Comment (oliverochman) [inline src/index.js]: change the baseURL back to https://glocal-news.herokuapp.com

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/Components/ListOfArticles.jsx (hunk 1 above)
Comment (raptorf1) [reply src/Components/ListOfArticles.jsx]: done

File: src/index.js (hunk 2 above)
Comment (raptorf1) [reply src/index.js]: done

**Review by oliverochman — APPROVED** [review]

**Pull request merged**